### ISSUE TITLE
State Website on Accessible Software - food for thought.

### ISSUE THREAD

**Issue by mel1877** [Sai83/accessibilityscore#2]
http://mn.gov/mnit/programs/accessibility/

**Comment by mel1877** [comment]
Perhaps we should consider classifying our scoring based on these categories.  I was thinking perhaps the sliders on the map.  We could exclude groups of data based on the type of disability.

Section 508 addresses accessibility for people who have: 

•Visual disabilities. 
•Hearing disabilities. 
•Motor disabilities. 
•Speech disabilities.

**Comment by mel1877** [comment]
The accessibility officer here at the State focuses more on making websites and applications useable for the handicap.  Although I think it would be ideal to follow these guidelines in developing our app.

**Comment by diavy** [comment]
I agree that classify the scoring based on categories is a good strategy. 
Correspondingly, we may figure out some special facility that is especially built for a certain type of disabilities. For example, a landlord with a "no pets" policy may be required to grant an exception to this rule and allow an individual who is blind to keep a guide dog in the residence.
  Then when we calculate the score, we should take that factor into consideration.

**Comment by fschilder** [comment]
great ideas! I guess we mostly were looking at motor disabilities which is
a good start though. Melinda, do you think there is data out there covering
other disabilities?

on another topic: would all of you be fine with mentioning their names in a
THomson Reuters internal post? I would like to post a short on the internal
web site (like facebook for TR exmployees).

Thanks,
Frank

**Comment by minmadic** [comment]
I'm cool with that Frank. :)

AJ

**Comment by fschilder** [comment]
If you're fine with it, could you quickly reply with your full name and a
short description (on the level of detail you would like to be described)?

Thanks,
Frank

**Comment by mel1877** [comment]
Melinda Miller - Sr. Management Analyst with the State of Minnesota MN.IT Services.

**Comment by diavy** [comment]
HI, Just want to share the following two official websites, which are authoritative information source about disabilities

One is MN
http://www.mndisability.gov/public/content.do?term_id=142&level=0 http://www.mndisability.gov/public/content.do?term_id=142&level=0

The other is federal:
https://www.disability.gov/ https://www.disability.gov/

Let’s see how we can integrate important features into our accessibility score for disabilities.
